cmd/tikv-ctl/src/cmd.rs

```diff
@@ -604,10 +604,22 @@ pub enum RaftCmd {
         // `regions` must be None when `all_regions` is present,
         // so we left `all_regions` unused.
         #[allow(dead_code)]
-        #[structopt(long, required_unless = "regions", conflicts_with = "regions")]
+        #[structopt(long, conflicts_with = "regions")]
         /// Print info for all regions
         all_regions: bool,
 
+        #[structopt(long, default_value = "")]
+        /// hex start key
+        start: String,
+
+        #[structopt(long, default_value = "")]
+        /// hex end key
+        end: String,
+
+        #[structopt(long, default_value = "0", max_values = MAX_REGION_LIMIT)]
```

Comment:
Why have an upper bound when the default behavior is infinite (limit=0)?

Reply:
Sry, to sync with pd-ctl, I'll set this to 16.

Reply:
I mean, if you allow user to print infinite amount of regions (limit = 0), then it doesn't make sense to limit the specific value of this field. I think it would be better to use 16 as default, and assign no restriction for max value.

Reply:
Get it, already modified.